pyproject.toml

```diff
@@ -108,4 +160,4 @@ fail_under = 63
 exclude_lines = [
     "pragma: no cover",
     "if __name__ == .__main__.:",
-]
```

Comment:
you can add the following line to the bottom of the `[tool.coverage.run]`  stanza to tell coverage to not count the __init__.py

```
omit = ['datagrepper/__init__.py']
```

Reply:
Should be finished